build/pch/pch.h

```diff
@@ -1,21 +1,24 @@
-//=============================================================================
-//  MuseScore
-//  Music Composition & Notation
-//
-//  Copyright (C) 2020 MuseScore BVBA and others
-//
-//  This program is free software; you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License version 2.
-//
-//  This program is distributed in the hope that it will be useful,
-//  but WITHOUT ANY WARRANTY; without even the implied warranty of
-//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//  GNU General Public License for more details.
-//
-//  You should have received a copy of the GNU General Public License
-//  along with this program; if not, write to the Free Software
-//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
-//=============================================================================
+/**
+ * // SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Something has gone wrong here. The formatter has added a second comment level when `//` is already sufficient.

Reply:
Ok, I realise what the problem is. The formatter thinks `//` is part of the comment, so you simply need to remove that and pass the license text in on its own. See my [comment above](https://github.com/musescore/MuseScore/pull/7960/files#discussion_r614445858).